static/app/components/events/eventEntries.tsx

```diff
@@ -461,28 +469,10 @@ const LatestEventNotAvailable = styled('div')`
   padding: ${space(2)} ${space(4)};
 `;
 
-const ErrorContainer = styled('div')`
-  /*
-  Remove border on adjacent context summary box.
-  Once that component uses emotion this will be harder.
-  */
-  & + .context-summary {
-    border-top: none;
-  }
-`;
-
 const BorderlessEventEntries = styled(EventEntries)`
-  & ${/* sc-selector */ DataSection} {
-    margin-left: 0 !important;
-    margin-right: 0 !important;
-    padding: ${space(3)} 0 0 0;
-  }
-  & ${/* sc-selector */ DataSection}:first-child {
-    padding-top: 0;
-    border-top: 0;
-  }
-  & ${/* sc-selector */ ErrorContainer} {
-    margin-bottom: ${space(2)};
+  padding: 0;
+  @media (max-width: ${p => p.theme.breakpoints.medium}) {
+    padding: 0;
   }
 `;
```

Comment:
Instead of adding another style override here to fix the alert's (which is wrapped inside a `StyledDataSection`) padding, I think it's better to refactor `DataSection` to have no horizontal padding at all, and then add the horizontal padding to its parent. 

Doing so gets rid of the style overrides, and results in a small visual change: the dividers between `DataSection`s no longer reach the edges. @robinrendle and I think this is fine.

**Issue Details – Before:**
<img width="1152" alt="Screen Shot 2022-07-21 at 10 50 39 AM" src="https://user-images.githubusercontent.com/44172267/180280728-ddfba819-50be-43b6-8dc1-b999148275c4.png">

**Issue Details – After:**
<img width="1152" alt="Screen Shot 2022-07-21 at 10 51 03 AM" src="https://user-images.githubusercontent.com/44172267/180280799-b8a35b4b-0d9b-4727-9e8f-37ec1775d273.png">